pkg/operation/common/types.go

```diff
@@ -337,6 +328,12 @@ const (
 	// ShootTaskDeployInfrastructure is a name for a Shoot's infrastructure deployment task.
 	ShootTaskDeployInfrastructure = "deployInfrastructure"
 
+	// ShootTaskDeployNetwork is a name for a Shoot's Network deployment task.
+	ShootTaskDeployNetwork = "deployNetwork"
+	// ShootNetworkTypeCalico is used to identify the calico network type
+	// TODO: remove once the new shoot API is introduced, value will be fetched from spec.networking.type
+	ShootNetworkTypeCalico = "calico"
```

Comment:
I'd not create a constant for this, let's rather 'hard-code' it where it's used.